README.md

```diff
@@ -11,49 +11,54 @@ Chris Herman
 Are your instructions for your new IKEA desk lacking in detail? Have you failed, time and time again, to attach the legs of your chairs the right way? Are you concerned about the large excess of screws you have once completing your new bed frame? IKEA hjälpare is here to be your helper!
 
 With IKEA hjälpare, you can:
+
```

Comment:
All of my README changes except the CircleCI badge actually came from my markdown lint plugin that automatically fixes lint errors when I save the file.

One of the big changes I made here was adding alternative text to the image links in case the image doesn't load

You can view the full list of markdown rules [here](https://github.com/DavidAnson/markdownlint/blob/v0.20.4/doc/Rules.md)

Reply:
That's cool, too!  A lint fixer?